src/cargo/ops/cargo_new.rs

```diff
@@ -435,12 +435,20 @@ mod tests {
     Ok(())
 }
 
+fn get_environment_variable(variables: &Vec<&str>)-> Option<String>{
+    variables.iter()
+    .filter_map(|var| env::var(var).ok())
+    .nth(0)
```

Comment:
For now Cargo aligns on the `.` for chains like this, and perhaps `.nth(0)` could get replaced with `next()`?